### PR TITLE
Use the same format for docstring

### PR DESCRIPTION
**Pull request by ki4070ma**
For https://github.com/appium/python-client/issues/387

* Reference
   * https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
   * https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
   * https://stackoverflow.com/questions/1137161/what-is-the-correct-way-to-document-a-kwargs-parameter
   * https://github.com/google/mobly

### CODE REVIEW

File: appium/webdriver/appium_service.py [hunk 1]
```diff
@@ -120,34 +120,35 @@ def _parse_host(args):
...
-        about possible arguments and their values.
-
-        :return:
-        subprocess.Popen instance. You can use Popen.communicate interface
-        or stderr/stdout properties of the instance
-        (stdout/stderr must not be set to None in such case)
-        in order to retrieve the actual process output.
+        Args:
+            kwargs:
+            `env`: Environment variables mapping. The default system environment,
```
Comment (mykola-mokhnach) [inline appium/webdriver/appium_service.py]: should argument types also be mentioned?

Comment (mykola-mokhnach) [reply appium/webdriver/appium_service.py]: same comment to above several docstrings below

File: appium/webdriver/extensions/screen_record.py [hunk 2]
```diff
@@ -20,88 +20,90 @@
...
-        - videoType: The format of the screen capture to be recorded.
-        Available formats: Execute `ffmpeg -codecs` in the terminal to see the list of supported video codecs.
-        'mjpeg' by default. (Since Appium 1.10.0)
-        - videoFps: The Frames Per Second rate of the recorded video. Change this value if the resulting video
-        is too slow or too fast. Defaults to 10. This can decrease the resulting file size.
-        - videoScale: The scaling value to apply. Read https://trac.ffmpeg.org/wiki/Scaling for possible values.
-        No scale is applied by default. (Since Appium 1.10.0)
-        - pixelFormat: Output pixel format. Run `ffmpeg -pix_fmts` to list possible values.
-        For Quicktime compatibility, set to "yuv420p" along with videoType: "libx264". (Since Appium 1.12.0)
+                iOS Specific:
```
[the remark this answers is not in the document]
Comment (ki4070ma) [reply appium/webdriver/extensions/screen_record.py]: Thanks for your comments.
Moved `iOS`, `Android` to each description.

File: appium/webdriver/extensions/images_comparison.py [hunk 3]
```diff
@@ -30,38 +30,40 @@ def match_images_features(self, base64_image1, base64_image2, **opts):
...
-                `points1` (dict)): The array of matching points on the first image. Each point is a dictionary
-                    with 'x' and 'y' keys
-                `rect1` (dict): The bounding rect for the `points1` array or a zero rect if not enough matching points
-                    were found. The rect is represented by a dictionary with 'x', 'y', 'width' and 'height' keys
-                `points2` (dict): The array of matching points on the second image. Each point is a dictionary
-                    with 'x' and 'y' keys
-                `rect2` (dict): The bounding rect for the `points2` array or a zero rect if not enough matching points
-                    were found. The rect is represented by a dictionary with 'x', 'y', 'width' and 'height' keys
+
+            `visualization` (bytes): base64-encoded content of PNG visualization of the current comparison
```
Comment (ki4070ma) [inline appium/webdriver/extensions/images_comparison.py]: Can remove '`'

File: appium/webdriver/extensions/applications.py [hunk 4]
```diff
@@ -46,20 +50,22 @@ def is_app_installed(self, bundle_id):
...
-         `timeout`: how much time to wait for the installation to complete.
-         60000ms by default.
-         `allowTestPackages`: whether to allow installation of packages marked
-         as test in the manifest. False by default
-         `useSdcard`: whether to use the SD card to install the app. False
-         by default
-         `grantPermissions`: whether to automatically grant application permissions
-         on Android 6+ after the installation completes. False by default
+        Args:
+            app_path: the local or remote path to the application to install
```
Comment (ki4070ma) [inline appium/webdriver/extensions/applications.py]: Add (str)

File: appium/webdriver/extensions/applications.py [hunk 5]
```diff
@@ -154,9 +172,9 @@ def app_strings(self, language=None, string_file=None):
         """Returns the application strings from the device for the specified
         language.
 
-        :Args:
-         - language - strings language code
-         - string_file - the name of the string file to query
+        Args:
```
Comment (ki4070ma) [inline appium/webdriver/extensions/applications.py]: Missing type

File: appium/webdriver/extensions/keyboard.py [hunk 6]
```diff
@@ -59,13 +71,17 @@ def keyevent(self, keycode, metastate=None):
...
-        :Args:
-         - keycode - the keycode to be sent to the device
-         - metastate - meta information about the keycode being sent
-         - flags - the set of key event flags
+        Android only. Possible keycodes can be found in http://developer.android.com/reference/android/view/KeyEvent.html.
+
+        Args:
+            keycode (int): the keycode to be sent to the device
+            metastate (:obj:`int`, optional): meta information about the keycode being sent
+            flags (:ojb: `int`, optional): the set of key event flags
```
Comment (ki4070ma) [inline appium/webdriver/extensions/keyboard.py]: Remove space

File: appium/webdriver/extensions/keyboard.py [hunk 7]
```diff
@@ -78,13 +94,17 @@ def press_keycode(self, keycode, metastate=None, flags=None):
...
-        """Sends a long press of keycode to the device. Android only. Possible keycodes can be
-        found in http://developer.android.com/reference/android/view/KeyEvent.html.
+        """Sends a long press of keycode to the device.
+
+        Android only. Possible keycodes can be found in http://developer.android.com/reference/android/view/KeyEvent.html.
+
+        Args:
+            keycode (int): the keycode to be sent to the device
+            metastate (:obj:`int`, optional): meta information about the keycode being sent
+            flags (:ojb: `int`, optional): the set of key event flags
```
Comment (ki4070ma) [inline appium/webdriver/extensions/keyboard.py]: Remove space

**Pull request merged**